Source/Core/Core/PowerPC/Interpreter/Interpreter_Branch.cpp

```diff
@@ -2,12 +2,12 @@
 // Licensed under GPLv2+
 // Refer to the license.txt file included.
 
-#include "Core/PowerPC/Interpreter/Interpreter.h"
```

Comment:
I _think_ the files containing the definition of what the cpp file defines should've been the first include in the file, for _$reasons_. Not sure if that can be formalized in a clang-format rule somewhere tho (or was this possible with a separating empty line between?)